.github/workflows/send.yml

```
@@ -14,21 +14,24 @@ jobs:
       - name: Build project
         run: yarn build
 
-      - name: Zip dist folder
-        uses: actions/upload-artifact@v4
-        with:
-          name: 'automatarium_dist'
-          path: ./frontend/dist/
+      - name: Zip dist folder to send
+        run: zip -r automatarium_frontend_dist.zip ./frontend/dist/
```

```suggestion
        run: cd frontend && zip -r ../automatarium_frontend_dist.zip ./dist/
```
The dist folder is nested inside a frontend folder when unzipped. You can change to this if you want remove the extra nesting. If you do, feel free to update the zip name as well.